examples/additional-rpc-namespace-in-cli/src/main.rs

```diff
@@ -31,6 +31,8 @@ struct MyRethCliExt;
 impl RethCliExt for MyRethCliExt {
     /// This tells the reth CLI to install the `txpool` rpc namespace via `RethCliTxpoolExt`
     type Node = RethCliTxpoolExt;
+    /// This is to install eventual custom components
+    type CustomComponents = RethDefaultComponents;
```

Comment:
The ergonomics with default associated types would improve a lot. Rather than that do you have any advise @mattsse ? 

I could not find a way to not explicitly set `RethDefaultComponents`.